R/data.R

```diff
@@ -0,0 +1,18 @@
+#' Ebola linelist data from Fang et al. (2016)
```

Comment:
roxygen needs to be generated

Reply:
I am a little confused in general about when to generate `roxygen2`. I suppose after each change which alters it is reasonable.

Reply:
yes. I agree its pretty boring. An approach I like the idea of but haven't done is to leave this as a CI job for main. The other option is to pin it into `pre-commit` so it always get done for you (we aren' using pre-commit here but maybe wee should)